cranelift/filetests/filetests/isa/x86/struct-arg.clif

```diff
@@ -0,0 +1,57 @@
+test compile
+set is_pic
+target x86_64
+
+function u0:0(i64 sarg(64)) -> i8 system_v {
```

Comment:
Can you add a second test with a function that takes two parameters, say, one i64 and one struct argument (in final position), please?